selfdrive/controls/controlsd.py

```diff
@@ -444,6 +438,12 @@ def data_sample(self):
         if not self.read_only:
           self.CI.init(self.CP, self.can_sock, self.pm.sock['sendcan'])
 
+          # Write CarParams for radard
+          cp_bytes = self.CP.to_bytes()
+          self.params.put("CarParams", cp_bytes)
+          put_nonblocking("CarParamsCache", cp_bytes)
+          put_nonblocking("CarParamsPersistent", cp_bytes)
+
```

Comment:
I'm not sure about this. I understand we want to update `radard` after `radarUnavailable` is set to `False` during the process of enabling radar tracks, but there may be some unknown implications with this change for other supported platforms.

With how sunnypilot/openpilot is set up currently, if `fwdRadar` is missing, the car simply would not get recognized. What if we set `radarUnavailable = False` under the targeted platforms in `_get_params` instead of modifying the flag in `init`?

https://github.com/sunnyhaibin/sunnypilot/blob/d79689b96779980aab3596184f19492ea67837f8/selfdrive/car/hyundai/interface.py#L68-L73

Reply:
Sounds good, let me change that. I wasn't sure about it too, but I didn't have a better solution, I was only able to verify that it works for me, on device. It's definitely possible that it would affect other platforms or runners.

Reply:
I added the changes, I'll need to confirm if it still works this way.